src/ir/comp.rs

```diff
@@ -891,6 +904,53 @@ impl CanDeriveDebug for CompInfo {
         can_derive_debug
     }
 }
+impl CanDeriveDefault for CompInfo {
+    type Extra = Option<Layout>;
+
+    fn can_derive_default(&self,
+                          ctx: &BindgenContext,
+                          layout: Option<Layout>)
+                          -> bool {
+        {
```

Comment:
nit: Why this block wrapping the full function? We should be able to just remove these braces and deindent it, right?